package/piksi_ins/piksi_ins.mk

```diff
@@ -17,13 +17,14 @@ endif # ! ($(BR2_BUILD_RELEASE_PROTECTED),y)
 
 $(info >>> Piksi INS is enabled, packaging with current image)
 
-PIKSI_INS_VERSION = 4e3c77a484bd4d3f0f53c4ab459b6966fc31c541
-PIKSI_INS_SITE = git@github.com:carnegieroboticsllc/piksi_ins.git
+PIKSI_INS_VERSION = master
```

Comment:
This should be tagged to a specific commit hash so master tags are idempotent (that is, not a moving target when rebuilding).

Reply:
agreed! I'll update when it works.

Reply:
I just updated to pin it, sorry, I'll stop poking at this now.